### PR TITLE
XW-3167 | Remove Old Page Header

### PR DESCRIPTION
**Pull request by hakubo**
https://wikia-inc.atlassian.net/browse/XW-3167

### CODE REVIEW

File: extensions/wikia/TemplateClassification/scripts/TemplateClassificationInCategory.js [hunk 1]
```diff
@@ -82,6 +82,7 @@ define('TemplateClassificationInCategory',
 
 				mw.messages.set(res.messages);
 
+				// TODO fix while working on MAIN-10625
```
Comment (WikiaTech) [inline extensions/wikia/TemplateClassification/scripts/TemplateClassificationInCategory.js:85]: INFO Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [ve.init.mw.ViewPageTarget.js#L194](https://github.com/Wikia/app/blob/acffb3cc279cb29fe82a65978990a9fda65eb6b5/extensions/VisualEditor/modules/ve-mw/init/targets/ve.init.mw.ViewPageTarget.js#L194): Remove the declaration of the unused '$firstHeading' variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [ve.init.mw.ViewPageTarget.js#L194](https://github.com/Wikia/app/blob/679802e22d8b988e8e6f365e45e1c1d4f07604dc/extensions/VisualEditor/modules/ve-mw/init/targets/ve.init.mw.ViewPageTarget.js#L194): Remove the declaration of the unused '$firstHeading' variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Comment by rogatty** [comment]
Ping @TK-999 and @macbre. We cleaned up some of your components too.

**Comment by hakubo** [comment]
LGTM - cannot approve as I have opened the PR

**Review by rybmat — APPROVED** [review]

**Pull request merged**